array.c

```diff
@@ -3224,12 +3290,13 @@ rb_ary_reverse_bang(VALUE ary)
 
 /*
  *  call-seq:
- *     ary.reverse    -> new_ary
- *
- *  Returns a new array containing +self+'s elements in reverse order.
+ *    array.reverse -> new_array
  *
- *     [ "a", "b", "c" ].reverse   #=> ["c", "b", "a"]
- *     [ 1 ].reverse               #=> [1]
+ *  Returns a new \Array whose elements (not copies) are in reverse order:
```

Comment:
I think the previous wording is better and more explicit.  If you must keep the new language, at least remove the ` (not copies)` part.